cirq-core/cirq/protocols/json_test_data/BooleanHamiltonian.json

```diff
@@ -1,15 +0,0 @@
-[
-  {
-    "cirq_type": "BooleanHamiltonian",
```

Comment:
@MichaelBroughton IIUC, We do need to preserve the de-serialization of objects that were previously serializable even though the object itself is being removed? So deserializing `BooleanHamiltonian` should now result in instantiating a new `BooleanHamiltonianGate` object?

Reply:
@tanujkhattar 
If Michael says to do so, how would I recover the qubits though? I think, if I am reading the code correctly, that the `BooleanHamiltonian` stores qubits, but the `BooleanHamiltonianGate` does not.

Reply:
The deserialization would return `BooleanHamiltonianGate(...).on(*qubits)`

Reply:
Done.